.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be checked in to source control, can you add it to the `.gitignore` and remove from this PR?